x/qgb/types/valset_confirm.go

```diff
@@ -0,0 +1,38 @@
+package types
+
+import (
+	sdk "github.com/cosmos/cosmos-sdk/types"
+	sdkerrors "github.com/cosmos/cosmos-sdk/types/errors"
+	stakingtypes "github.com/cosmos/cosmos-sdk/x/staking/types"
+	"github.com/ethereum/go-ethereum/common"
+)
+
+var _ sdk.Msg = &MsgValsetConfirm{}
+
+// GetSigners defines whose signature is required.
+func (msg *MsgValsetConfirm) GetSigners() []sdk.AccAddress {
+	// TODO: figure out how to convert between AccAddress and ValAddress properly
+	acc, err := sdk.AccAddressFromBech32(msg.Orchestrator)
+	if err != nil {
+		panic(err)
+	}
+
+	return []sdk.AccAddress{acc}
+}
+
+// ValidateBasic performs stateless checks.
+func (msg *MsgValsetConfirm) ValidateBasic() (err error) {
+	if _, err = sdk.AccAddressFromBech32(msg.Orchestrator); err != nil {
+		return sdkerrors.Wrap(sdkerrors.ErrInvalidAddress, msg.Orchestrator)
+	}
+	if !common.IsHexAddress(msg.EthAddress) {
+		return sdkerrors.Wrap(stakingtypes.ErrEthAddressNotHex, "ethereum address")
+	}
+	return nil
+}
+
+// Type should return the action.
+func (msg *MsgValsetConfirm) Type() string { return "/qgb.MsgValsetConfirm" }
```

Comment:
[question] are there naming conventions around the return value? `/qgb.MsgValsetConfirm` isn't consistent with the previous type `data_commitment_confirm` but it's not clear to me which naming conventions are preferred for this value.

Reply:
There is an issue to decide on conventions for this and also on the cosmos-sdk side:
https://github.com/celestiaorg/celestia-app/issues/303 and https://github.com/celestiaorg/cosmos-sdk/issues/136